src/components/GraphiQLPlain/index.ts

```diff
@@ -0,0 +1,2 @@
+export { default } from "./GraphiQL";
```

Comment:
**suggestion**: what about using a named export instead of the default one?

Reply:
@krzysztofzuraw I agree with you but here I made it how it's done for all other components at `src/components` in this codebase for consistency